catalogs/mississippi-s1/ingest_s1.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+import argparse
+from datetime import date, time, datetime
+import logging
+import sys
+
+import boto3
+from dateutil import rrule, relativedelta
+
+from request_builders import (
+    get_sentinel_hub_session,
+    search_sentinelhub_s1,
+    create_batch_request,
+    analyze_batch_request,
+    initiate_batch_request,
+    check_batch_status,
+)
+from stac_utils.s3_io import register_s3_io
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+def make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
The docker container forwards environment variables for oauth id and secret: https://github.com/azavea/noaa-flood-mapping/blob/master/docker-compose.yml#L8-L9

We could keep the arguments here, but perhaps tweak them so they pick up defaults from `os.environ`.